Assets/Scripts/CaptureSceneManager.cs

```diff
@@ -77,12 +83,21 @@ void Start()
 
         correctStatusFront = GameObject.Find("CorrectStatusTextFront").GetComponent<TextMeshProUGUI>();
         correctStatusShadow = GameObject.Find("CorrectStatusTextShadow").GetComponent<TextMeshProUGUI>();
-        
+
+        // after getting button references, we deactivate the canvas
+        questionAnswerCanvas.gameObject.SetActive(false);
+
         StartCoroutine(GetMathProblems());
+
     }
 
     void Update()
     {
+        if (captureViewBeastie != null && !warningInactive)

```

Comment:
minor: It might be slightly easier to read if we name the variables using positives. So instead of calling the variable `warningInactive`, we could call the variable `warningActive` or `warningIsActive`. Then this line would be:

`if (captureViewBeastie != null && warningIsActive)`